isso/js/app/templates/postbox.js

```diff
@@ -1,8 +1,10 @@
 var html = function (globals) {
   var i18n = globals.i18n;
+  var conf = globals.conf;
   var author = globals.author;
   var email = globals.email;
   var website = globals.website;
+  var notify = conf["reply-notifications-default-enabled"] ? " checked" : '';
```

Comment:
Maybe someone has a nicer suggestion than this ternary. But not a big deal.

Reply:
Would it be good to change it to be a boolean (just `conf["reply-notifications-default-enabled"]`), and then put the ternary inside with the other template code?

Reply:
Eh, it's nitpicking and I'm not even qualified to comment on any Javascript styles. I was hoping for someone to have a better idea, but it's not important. Leave it as-is.